dags/index_and_normalise_notice_worker.py

```diff
@@ -11,7 +11,14 @@
 from airflow.operators.python import get_current_context
 
 from dags import DEFAULT_DAG_ARGUMENTS
+from ted_sws.event_manager.adapters.event_log_decorator import event_log
+from ted_sws.event_manager.adapters.event_logger import EventLogger
+from ted_sws.event_manager.model.event_message import NoticeEventMessage
+from datetime import datetime
+from ted_sws.event_manager.services.logger_from_context import get_logger_from_dag_context, \
+    get_dag_args_from_context
 
+DAG_KEY = f"index_and_normalise_notice_worker_{datetime.now().isoformat()}"
```

Comment:
This line executed once, I can't guarantee that this line will behave like DAG_RUN_ID (which is generated by airflow) and is different at each DAG run.

Reply:
this line is meant to keep connection between dag tasks and decorator logging

Reply:
the dag run_id is added by get_dag_args_from_context(context) service